webapp/packages/plugin-authentication-administration/src/Administration/ServerConfiguration/ServerConfigurationAuthenticationBootstrap.ts

```diff
@@ -85,6 +86,7 @@ export class ServerConfigurationAuthenticationBootstrap extends Bootstrap {
       return validation.invalidate();
     }
 
+    await this.passwordPolicyResource.load();
```

Comment:
No, we need to keep in mind that resources can be invalidated at any time and `load` executed only once on app start, we also don't need this resource to be loaded in app starting process because we only need this data in certain forms